spec/features/find/search/editing_a_search_spec.rb

```diff
@@ -32,7 +32,7 @@ def when_i_execute_a_valid_search
     and_i_click_continue
     and_i_select_the_primary_subject_checkbox
     and_i_click_continue
-    and_i_select_the_not_yet_visa_status_radio_button
+    and_i_choose_yes_to_visa_sponsorship
     and_i_click_find_courses
```

Comment:
should we add a step `then_the_visa_filter_should_be_checked` or something related?

Reply:
Great idea. 

Done here: https://github.com/DFE-Digital/publish-teacher-training/pull/3906/commits/779374366a8669ad53bf8b95f691bc3c7af780ae